sharepoint/sharepoint-server-ps/sharepoint-server/Switch-SPCertificate.md

```diff
@@ -37,13 +35,7 @@ This example replaces the assignments of the "Team Sites Certificate (2020)" cer
 ## PARAMETERS
 
 ### -AssignmentCollection
-Manages objects for the purpose of proper disposal.
-Use of objects, such as SPWeb or SPSite, can use large amounts of memory and use of these objects in Windows PowerShell scripts requires proper memory management.
-Using the SPAssignment object, you can assign objects to a variable and dispose of the objects after they are needed to free up memory.
-When SPWeb, SPSite, or SPSiteAdministration objects are used, the objects are automatically disposed of if an assignment collection or the Global parameter is not used.
-
-When the Global parameter is used, all objects are contained in the global store.
-If objects are not immediately used, or disposed of by using the Stop-SPAssignment command, an out-of-memory scenario can occur.
+{{ Fill AssignmentCollection Description }}
```

Comment:
What is the reason to remove the description? In case it's not valid, a valid one should be use instead an not a fill placeholder